test/core/derivatives/derivatives_test.py

```diff
@@ -46,26 +47,42 @@
     problem.make_id() for problem in CONVOLUTION_TRANSPOSED_FAIL_PROBLEMS
 ]
 
+RNN_PROBLEMS = make_test_problems(RNN_SETTINGS)
+RNN_IDS = [problem.make_id() for problem in RNN_PROBLEMS]
 
-@pytest.mark.parametrize("problem", NO_LOSS_PROBLEMS, ids=NO_LOSS_IDS)
-def test_jac_mat_prod(problem, V=3):
+
+# selecting only every second rnn problem (only cpu problems)
+# because torch is not able to calculate this function on CUDA
```

Comment:
Is the selection statement true? The code looks as if the `autograd` computation is run with a backend other than `cudnn`

Reply:
This was an old comment. Deleted.